CHANGES.rst

```diff
@@ -31,6 +31,8 @@ master_background
 pipeline
 --------
 
+- Added FGS_IMAGE to the exposure types to be processed in calwebb_image2.py [#4421]
```

Comment:
FGS_IMAGE exposures are already processed in calwebb_image2. What's actually changed is that the ``resample`` step is now applied to them. So "... exposure types to apply resampling in calwebb_image2.py".